### ISSUE TITLE
Super Smash Bros. Ultimates keeps having issues with suttering post hades. 

### ISSUE THREAD

**Issue by pkNurseJoy** [yuzu-emu/yuzu#6893]
I and many others have been experience a huge amount of stuttering and inconsistent frametimes for Super Smash Bros. Ultimate ever since Hades released. I have tested in both vulkan and OpenGl and no matter what I and many others seem to experience this. I have no idea why it happens, I tried having discord on, off, no HW acceleration, etc.. and it does not make a difference. 

here's my log if it matters. 
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7020720/yuzu_log.txt)
I really hope you guys can figure out what is causing this. Because pre-hades it was totally fine.

**Comment by v1993** [comment]
A wild guess - hades have invalidated old shader caches. Could this be the reason?

**Comment by FernandoS27** [comment]
what settings have you tried playing with?

have you tried turning of the Garbage Collector?

**Comment by pkNurseJoy** [comment]
> A wild guess - hades have invalidated old shader caches. Could this be the reason?

I wiped out my shaders before upgrading to hades.

**Comment by pkNurseJoy** [comment]
> what settings have you tried playing with?
> 
> have you tried turning off the Garbage Collector?

No I have not. You think it would have any effect though?

**Comment by FernandoS27** [comment]
> > what settings have you tried playing with?
> > have you tried turning off the Garbage Collector?
> 
> No I have not. You think it would have any effect though?
> 
> Just tested, it made no difference.

ok let's try thinking out of the box. Try disabling NVIDIA G-Sync in NVIDIA Cotrol Panel.

**Comment by redpanda404** [comment]
been having the same problem/s as nurse. almost feels like hades just went opposite of its intended goal, hope this can get looked into because before hades smash ultimate was running fine most of the time for me.

**Comment by campagne999** [comment]
similar issues with a bit more crashing as of late in normal 1v1 play.

**Comment by pkNurseJoy** [comment]
> > > what settings have you tried playing with?
> > > have you tried turning off the Garbage Collector?
> > 
> > 
> > No I have not. You think it would have any effect though?
> > Just tested, it made no difference.
> 
> ok let's try thinking out of the box. Try disabling NVIDIA G-Sync in NVIDIA Cotrol Panel.

That does nothing either. 

Just as I suspected

**Comment by FernandoS27** [comment]
are you guys using an external display?

**Comment by pkNurseJoy** [comment]
> are you guys using an external display?

No, I am on a desktop. Using dual 360hz monitors. I removed one of them and the issue is still there

**Comment by Struggleton** [comment]
I've had similar issues on Smash Ultimate. I'm using two monitors - one 144hz (Viewsonic XG2401) and a 75hz (
Acer KA220HQ.) 
Not sure if I can provide anymore infos other than it seems to happen on both Vulkan and OpenGL. I can post a video I recorded on the host machine while playing a session with the microprofile shown on screen @FernandoS27

**Comment by FernandoS27** [comment]
hmn

**Comment by gidoly** [comment]
is this fixed?
